guides/micronaut-json-schema-generator/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Micronaut JSON Schema",
+  "intro": "Generate Java objects from a JSON schema at compilation thanks to Micronaut JSON Schema Generator.",
```

Comment:
Generate Java <s>objects</s> classes from a JSON schema at compilation <s>thanks</s> using the Micronaut JSON Schema <s>Generator</s> generator.